core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -1894,4 +1898,35 @@ public static String getDataMapStorageProvider() {
     }
     return provider.toUpperCase();
   }
+
+  /**
+   * This method validates the duration for carbon cache expiration
+   */
+  private void validateDurationForCacheExpiration() {
+    String defaultValue = Long.toString(
+        CarbonCommonConstants.CARBON_LRU_CACHE_EXPIRATION_DURATION_IN_MINUTES_DEFAULT);
+    long duration;
+    try {
+      duration = Long.parseLong(
+          carbonProperties.getProperty(CARBON_LRU_CACHE_EXPIRATION_DURATION_IN_MINUTES));
+      if (duration < 0 || duration == 0) {
+        LOGGER.info("using default value of carbon.lru.cache.expiration.duration.in.minutes = "
+            + CarbonCommonConstants.CARBON_LRU_CACHE_EXPIRATION_DURATION_IN_MINUTES_DEFAULT);
+        carbonProperties
+            .setProperty(CarbonCommonConstants.CARBON_LOCAL_DICTIONARY_SIZE_THRESHOLD_IN_MB,
+                defaultValue);
+      } else {
+        LOGGER.info("using carbon.lru.cache.expiration.duration.in.minutes = " + duration);
+        carbonProperties
+            .setProperty(CARBON_LRU_CACHE_EXPIRATION_DURATION_IN_MINUTES,
+                Long.toString(duration));
+      }
+    } catch (Exception e) {
```

Comment:
In this method, only NumberFormatException will be thrown, i dont see any other type of exception. If not instead of generalised exception, use  NumberFormatException.